### DISCUSSION TITLE
Is there a BibTeX template for citation?

### DISCUSSION THREAD

**Discussion by beckstev** [colour-science/colour#745] [Q&A] [answered]
May you consider to provide a bibtex citation, so that we can cite this greate package.

**Comment by beckstev** [comment]
I have found the following link, I assume this is your preferred way.
https://zenodo.org/record/3627408#.XrEOnJlCSM8

You could add this to the repo as well, so that users can find it more easily.

**Comment by thomasmansencal** [comment, marked as answer]
Hi @beckstev,

If you follow the [DOI badge link](https://zenodo.org/record/3627408) in the [README.rst](https://github.com/colour-science/colour/blob/develop/README.rst) file:

![image](https://user-images.githubusercontent.com/99779/81041927-206e6100-8f03-11ea-9bb9-55f021e20710.png)

You will land on the Zenodo record, from which it is possible to get the BibTex citation:

![image](https://user-images.githubusercontent.com/99779/81041871-fcab1b00-8f02-11ea-97ff-3535e454cad0.png)

Cheers,

Thomas

**Comment by beckstev** [comment]
Thanks, havent seen the badge.

**Comment by thomasmansencal** [comment]
No worries it is not necessarily obvious! There is also a link on the website which is not obvious either: 
![image](https://user-images.githubusercontent.com/99779/81042197-b3a79680-8f03-11ea-8e04-3b17351a09c1.png)

We might add something directly on the landing page actually!

Thanks for the feedback.

Thomas
